<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ComputeRadianceDistance</class>
 <widget class="QDialog" name="Dialog_ComputeRadianceDistance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compute Distance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="list_maps_1"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Position attribute :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_positionAttribute_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Normal attribute :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_normalAttribute_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Distance attribute</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Existing attribute :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combo_distanceAttribute_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>New attribute :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="distanceAttributeName_1"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="list_maps_2"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Position attribute :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_positionAttribute_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Normal attribute :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_normalAttribute_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Distance attribute</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Existing attribute :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combo_distanceAttribute_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>New attribute :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="distanceAttributeName_2"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="enableVBO">
     <property name="text">
      <string>Create VBOs</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_ok</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_ComputeRadianceDistance</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_ComputeRadianceDistance</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
